### ISSUE TITLE
Logging support on server side

### ISSUE THREAD

**Issue by VikramTiwari** [VikramTiwari/webstack#37]
Follow the approach of using a logger and transports so that both can be maintained and extended individually.

**Comment by avipatel91** [comment]
How about using [winston](https://github.com/winstonjs/winston) for this?

**Comment by VikramTiwari** [comment]
Yeah! That's the plan. Just wanna do some research into what would be the best way to integrate with current structure.
